kanban-board/src/main/java/io/spine/examples/kanban/server/card/CardStatusChange.java

```diff
@@ -34,147 +34,147 @@
 import static io.spine.util.Exceptions.newIllegalStateException;
 
 /**
- * Generates task status change events.
+ * Generates card status change card.event.
  *
- * @see TaskAggregate#on(TaskMoved)
+ * @see CardAggregate#on(io.spine.examples.kanban.card.event.CardMoved)
  */
-final class TaskStatusChange {
+final class CardStatusChange {
 
     /** Prevents instantiation. */
-    private TaskStatusChange() {
+    private CardStatusChange() {
     }
 
     /**
-     * Generates a task status change event wrapped in either of four upon moving a task in columns
+     * Generates a card status change event wrapped in either of four upon moving a card in columns
      * on a Kanban board.
      *
      * @param event
-     *         an event from Kanban board about a task being moved from one column to another
-     * @return either of four task events:
-     *         {@link TaskReopened}, {@link WorkSuspended}, {@link WorkStarted}, {@link TaskDone}
+     *         an event from Kanban board about a card being moved from one column to another
+     * @return either of four card events:
+     *         {@link io.spine.examples.kanban.card.event.CardReopened}, {@link WorkSuspended}, {@link WorkStarted}, {@link io.spine.examples.kanban.card.event.CardDone}
```

Comment:
Absolute imports can be discarded here for consistency with other links.
```suggestion
     *         {@link CardReopened}, {@link WorkSuspended}, {@link WorkStarted}, {@link CardDone}
```